src/main/java/com/zaxxer/nuprocess/linux/EpollEvent.java

```diff
@@ -106,6 +100,25 @@ int getFieldOffset(String field)
          return Arrays.asList("events", "data");
       }
 
+      /**
+       * Uses the OS architecture to reproduce the following logic from the epoll header:
+       * <code><pre>
+       * #ifdef __x86_64__
+       * #define EPOLL_PACKED __attribute__((packed))
+       * #else
+       * #define EPOLL_PACKED
+       * #endif
+       * </pre></code>
+       *
+       * On x86-64 (amd64) platforms, {@code ALIGN_NONE} is used (to emulate {@code __attribute__((packed))}),
+       * and on all other platforms {@code ALIGN_GNUC} is used.
+       */
+      static int detectAlignment() {
+         String arch = System.getProperty("os.arch");
+
+         return "amd64".equals(arch) || "x86-64".equals(arch) ? ALIGN_NONE : ALIGN_GNUC;
```

Comment:
Based on JNA's [getCanonicalArchitecture](https://github.com/java-native-access/jna/blob/master/src/com/sun/jna/Platform.java#L235-L260) this probably needs to test for "x86_64" too